core/src/main/java/rdx/works/core/preferences/PreferencesManager.kt

```diff
@@ -132,6 +132,17 @@ class PreferencesManager @Inject constructor(
         }
     }
 
+    val isLinkConnectionStatusIndicatorEnabled: Flow<Boolean> = dataStore.data
+        .map { preferences ->
+            preferences[KEY_LINK_CONNECTION_STATUS_INDICATOR] ?: true
```

Comment:
I would change the default value to false. Better for us to opt in. Less intrusive

Reply:
sure! changes pushed.